src/plugins/auto/plugin.cpp

```diff
@@ -806,7 +799,45 @@ std::string MultiDeviceInferencePlugin::GetDeviceList(const std::map<std::string
 
     return allDevices;
 }
-
+std::vector<std::string > MultiDeviceInferencePlugin::ParsePrioritiesDevices(const std::string& priorities, const char separator) {
+    std::vector<std::string> devices;
+    std::string::size_type pos = 0;
+    std::string::size_type endpos = 0;
+    auto isAvailableDevice = [&](std::string& deviceName) -> bool {
+        if (deviceName.empty())
+            return false;
+        auto realDevName = deviceName[0] != '-' ? deviceName : deviceName.substr(1);
+        if (realDevName.empty()) {
+            return false;
+        }
+        realDevName = DeviceIDParser(realDevName).getDeviceName();
+        std::string::size_type realEndPos = 0;
+        if ((realEndPos = realDevName.find('(')) != std::string::npos) {
+            realDevName = realDevName.substr(0, realEndPos);
+        }
+        if (_availableDevices.end() ==
+            std::find_if(_availableDevices.begin(), _availableDevices.end(), [&realDevName] (const std::string& device) {
+                return device.find(realDevName) != std::string::npos;
```

Comment:
something minor: or maybe don;t handle "(", just return realDevName.find(device)  != npos, is this what you expected?